### PR TITLE
[WIP] 8.0 web list html widget

### PR DESCRIPTION
**Pull request by damdam-s**

### CODE REVIEW

File: web_list_html_widget/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    "name": "HTML widget for list/tree views",
+    "version": "0.1",
```
Comment (yvaucher) [inline web_list_html_widget/__openerp__.py]: Put full version here: `8.0.0.1`

File: web_list_html_widget/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    "depends": ["web"],
+    'author': "initOS GmbH & Co. KG, Odoo Community Association (OCA), "
+              "Camptocamp",
+    "category": "",
```
Comment (yvaucher) [inline web_list_html_widget/__openerp__.py]: No category to provide?

File: web_list_html_widget/README.rst [hunk 3]
```diff
@@ -0,0 +1,45 @@
+Widget Html rendering in list views
```
Comment (yvaucher) [inline web_list_html_widget/README.rst]: Missing the AGPL logo on top

File: web_list_html_widget/README.rst [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+===================================
+
+This module allows to use a ``widget="html"`` for fields in list(tree) views.
+
+*WARNING* The content of the field will be rendered without HTML escaping and without any other security measure.
```
Comment (gurneyalex) [inline web_list_html_widget/README.rst:11]: WAT?

File: web_list_html_widget/README.rst [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+Features
+========
+
+* Add widget 'html' in a Html field on a tree view
+
+
```
Comment (yvaucher) [inline web_list_html_widget/README.rst:21]: Missing the try on runbot somewhere here, have a look at the template:

https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

File: web_list_html_widget/__openerp__.py (hunk 1 above)
Comment (pedrobaeza) [inline web_list_html_widget/__openerp__.py]: Please bump it to 8.0.1.0.0 as this is the first 8.0 version

File: web_list_html_widget/README.rst (hunk 4 above)
Comment (damdam-s) [inline web_list_html_widget/README.rst:11]: Actually, I didn't succeeded in breaking Odoo security with this widget. Maybe someone can confirm/invalidate this ?
FYI, I made some tests from this site : https://www.owasp.org/index.php/Testing_for_Cross_site_scripting

**Comment by damdam-s** [comment]
up ? anyone ?

File: web_list_html_widget/i18n/web_list_html_widget.pot [hunk 6]
```diff
@@ -0,0 +1,16 @@
+# Translation of OpenERP Server.
```
Comment (pedrobaeza) [inline web_list_html_widget/i18n/web_list_html_widget.pot:1]: Remove this file, as it's not needed anymore

File: web_list_html_widget/__init__.py [hunk 7]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```
Comment (pedrobaeza) [inline web_list_html_widget/__init__.py:1]: Please shorten headers as in https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py

**Comment by pedrobaeza** [comment]
To be consistent with other names, please call it _web_widget_html_treeview_

**Comment by damdam-s** [comment]
@pedrobaeza ok for the two first comments.
For the last, as it's a migration from v8 do we really need (have to) to change the module name ?
No problem if I have to change it but more opinions on this are welcome

File: web_list_html_widget/README.rst (hunk 4 above)
Comment (dreispt) [inline web_list_html_widget/README.rst:11]: HTML fields support a `sanitize` attribute. You could recommend (or enforce?) to use only HTML fields with the `sanitize=True` attribute set.

**Comment by yvaucher** [comment]
I agree with @pedrobaeza 
It would be more consistent with other module names:

```
web_widget_color 
web_widget_datepicker_options   
web_widget_float_formula    
web_widget_many2many_tags_multi_selection   
web_widget_text_markdown    
web_widget_x2many_2d_matrix
```

Plus it has no model definition that could make a migration from one to the other really tricky.

**Comment by pedrobaeza** [comment]
There are still comments unattended and the module renaming. Are you going to finish this?

**Comment by jcdrubay** [comment]
It seems like this is a duplicate with #902

**Pull request closed without merging**